infra/gcp/terraform/k8s-infra-oci-proxy/main.tf

```diff
@@ -35,4 +35,6 @@ module "oci-proxy" {
   // Manually created. Monitoring channels can't be created with Terraform.
   // See: https://github.com/hashicorp/terraform-provider-google/issues/1134
   notification_channel_id = "3237876589275698022"
+  // TEMPORARY: re-using existing cert for new-type LB
+  ssl_certificates = ["k8s-infra-oci-proxy-cert-771cbf9e"]
```

Comment:
When we go to tear down the lb-http stuff we'll add something like https://github.com/kubernetes/k8s.io/blob/main/infra/gcp/terraform/k8s-infra-porche-prod/certificate.tf

And migrate to a cert managed within the module.

I think it's OK to do that later though, and I'm focused on getting this up in staging so we can confirm that migrating to envoy based GCLB with outlier_detection will meet our failover requirements versus switching to some other approach.